### PR TITLE
 OP-780 - OH GUI make spring use xxx ACT (rebased) 

### PR DESCRIPTION
**Pull request by goto-eof**
See OP-780.
Paired with https://github.com/informatici/openhospital-core/pull/767.

### CODE REVIEW

File: src/main/java/org/isf/admission/gui/AdmissionBrowser.java [hunk 1]
```diff
@@ -1830,7 +1830,7 @@ private JButton getSaveButton() {
 					admission.setType("N");
 				}
 
-				admission.setDeleted("N");
+				admission.setActive(1);
```
Comment (mwithi) [inline src/main/java/org/isf/admission/gui/AdmissionBrowser.java:1833]: Is this command really needed now?

**Review by mwithi — CHANGES_REQUESTED** [review]
The PR will not compile after the core merge because more methods have been renamed

File: src/main/java/org/isf/admission/gui/AdmissionBrowser.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (goto-eof) [reply src/main/java/org/isf/admission/gui/AdmissionBrowser.java]: done

File: src/main/java/org/isf/supplier/gui/SupplierBrowser.java [hunk 2]
```diff
@@ -337,7 +337,7 @@ public Object getValueAt(int r, int c) {
 			} else if (c == 7) {
 				return sup.getSupNote();
 			} else if (c == 8) {
-				return sup.getSupDeleted().equals('Y');
+				return sup.getActive().equals(0);
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/supplier/gui/SupplierBrowser.java:340]: Doesn't `getActive()` return an `int`?   So why isn't the test `return sup.getActive() == 1`?

**Pull request merged**